certified-connectors/SeeBotRun Link/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1353 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SeeBotRun - Link",
+    "description": "Create links dynamically for your SeeBotRun Link account.",
```

Comment:
@rwwmatt Can you provide some more information for the description.

Documentation - [Description of connector](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#description)

Reply:
Yep, no problem.  Would something like this suffice?  

> Create and manage short links dynamically for your SeeBotRun Link account, using provided or custom domains.